Hudl.Mjolnir.Attributes/CommandAttribute.cs

```diff
@@ -15,10 +15,23 @@ public sealed class CommandAttribute : Attribute
         private readonly string _breakerKey;
         private readonly string _poolKey;
         private readonly int _timeout;
+        private readonly bool _ignoreTimeout;
 
         // See Mjolnir's Command constructors.
         public CommandAttribute(string group, string isolationKey, int timeout = DefaultTimeout) : this(group, isolationKey, isolationKey, timeout) { }
 
+        public CommandAttribute(string group, string isolationKey, bool ignoreTimeout)
```

Comment:
Yeah, I think that's the way to go - it'll more align the tests with how the library is supposed to behave (i.e. driven by the config value). If we *have* to go this route, I think we'd want this constructor to be `internal` so that it's not part of the public API.